.gitignore

```diff
@@ -26,3 +26,4 @@ chrome-user-data
 .idea
 *.iml
 .vscode
+bundle.js
```

Comment:
Can you move this up with the other line where we reference examples, and then precede this with `examples/**/` so we don't ignore anything else.